Completion/Unix/Command/_find

```diff
@@ -37,6 +37,8 @@ case $variant in
     args+=(
       '*-Bmin:birth time (minutes)'
       '*-Bnewer:file to compare (birth time):_files'
+      '*-newer'{a,B,c,m}{a,B,c,m}'[succeed if timestamp 1 is newer than timestamp 2 of the given file]:reference file:_files'
+      '*-newer'{a,B,c,m}t'[succeed if timestamp of given file is newer than timestamp]:timestamp: '
```

Comment:
"given" is what's in find(1)'s argv, and due to limited horizontal space I'd like to shorten "succeed if" which would otherwise be on almost every line, so:

```suggestion
      '*-newerat[if access time is newer than given timestamp]:timestamp: '
      '*-newerBt[if birth time is newer than given timestamp]:timestamp: '
      '*-newerct[if creation time is newer than given timestamp]:timestamp: '
      '*-newermt[if modification time is newer than given timestamp]:timestamp: '
```